flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/DataStream.scala

```diff
@@ -44,7 +44,14 @@ class DataStream[T](javaStream: JavaStream[T]) {
   /**
    * Gets the underlying java DataStream object.
    */
-  def getJavaStream: JavaStream[T] = javaStream
+  private[flink] def getJavaStream: JavaStream[T] = javaStream
```

Comment:
Why not simply give the variable `javaStream` the access modifier `private[flink]`.